src/extensions/bigbang/manifests.go

```diff
@@ -63,12 +63,27 @@ kyvernoPolicies:
           - zarf # don't have Kyverno prevent Zarf from doing zarf things
           `
 
-func manifestZarfCredentials(version string) corev1.Secret {
+const bbV2ZarfYoloModeCredentialsValues = `
+# -- Big Bang v2 Kyverno Support
+kyvernoPolicies:
+  values:
+    exclude:
+      any:
+      - resources:
+          namespaces:
+          - zarf # don't have Kyverno prevent Zarf from doing zarf things
```

Comment:
Thanks for the PR! Technically if you were running YOLO mode you shouldn't need this either since the Zarf agent would not be running in cluster.

Reply:
dropped these values in lieu of setting `registryCredentials` instead.